### PR TITLE
Adding r-matchthem from CRAN

### PR DESCRIPTION
**Pull request by dillonroach**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-matchthem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    - r-survey
+
+test:
+  commands:
+    - $R -e "library('MatchThem')"           # [not win]
+    - "\"%R%\" -e \"library('MatchThem')\""  # [win]
+
+about:
+  home: https://github.com/FarhadPishgar/MatchThem
+  license: GPL-2.0-only
```
Comment (mfansler) [inline recipes/r-matchthem/meta.yaml:53]: DESCRIPTION shows `GPL (>= 2)`, so

```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-matchthem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  summary: Provides essential tools for the pre-processing techniques of matching and weighting
+    multiply imputed datasets. The package includes functions for matching within and
+    across multiply imputed datasets using various methods, estimating weights for units
+    in the imputed datasets using multiple weighting methods, calculating causal effect
+    estimates in each matched or weighted dataset using parametric or non-parametric
+    statistical models, and pooling the resulting estimates according to Rubin's rules
+    (please see <https://journal.r-project.org/archive/2021/RJ-2021-073/> for more details).
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```
Comment (mfansler) [inline recipes/r-matchthem/meta.yaml:63]: Expand

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting!

Minor adjust, then should be good to merge.

File: recipes/r-matchthem/meta.yaml (hunk 1 above)
Comment (dillonroach) [reply recipes/r-matchthem/meta.yaml]: Good catch!, updated.

**Comment by mfansler** [comment]
Keep noarch and let win fail.

**Review by mfansler — APPROVED** [review]

**Pull request merged**